HalfedgeDS/include/CGAL/HalfedgeDS_list.h

```diff
@@ -385,7 +385,11 @@ class HalfedgeDS_list
         // halfedges, and f faces. The reservation sizes are a hint for
         // optimizing storage allocation. They are not used here.
 
-    ~HalfedgeDS_list() { clear(); }
+    ~HalfedgeDS_list() noexcept {
+      try {
+        clear();
+      } catch (...) {}
+    }
```

Comment:
How is it possible that a `clear()` can trigger? What does it do?!

Reply:
... Oh... is it about assertions in the `clear()` functions, that can trigger?

Reply:
precisely they are all from CGAL_assertion. I didn't think the overhead of this code in release builds was significant, to warrant using `CGAL_assertion_code`